src/main/java/org/futureCollars/lesson10/task2/PeselValidator.java

```diff
@@ -0,0 +1,18 @@
+package org.futureCollars.lesson10.task2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PeselValidator {
+
+    public static void main(String[] args) {
```

Comment:
no ale przenieś to może do jakiejś metody o nazwie isPeselValid albo isValid i dopisz do tego jakieś testy czy coś :)